tests/Feature/AddressLookupTest.php

```diff
@@ -76,8 +76,8 @@
         ->assertSeeInOrder([
             '<?xml version="1.0" encoding="UTF-8"?>',
             '<Response>',
-            '<Say voice="alice" language="en-US">searching meeting information for UkEEma, WA 98901, USA</Say>',
-            '<Redirect method="GET">meeting-search.php?Latitude=46.6140435&amp;Longitude=-120.4454218</Redirect>',
+            '<Say voice="alice" language="en-US">searching meeting information for Skwim, WA 98382, USA</Say>',
+            '<Redirect method="GET">meeting-search.php?Latitude=48.0772936&amp;Longitude=-123.1319107</Redirect>',
```

Comment:
can you update this to?

`<Redirect method="GET">meeting-search.php?Latitude=48.0217704&amp;Longitude=-123.1273844</Redirect></Response>`